templates/basic/gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 node_modules
 dist
+coverage
```

Comment:
```suggestion
# Jest test coverage reports
coverage
```

Could we add a comment here? I'm planning on adding comments to other portions of this file similar to #864